qvarn/backends/postgresql.py

```diff
@@ -335,9 +372,30 @@ def init(self):
             async with conn.begin():
                 await conn.execute(table.insert().values(id=row_id, revision=revision, data=data, search=search))
                 await self._update_aux_tables(conn, resource_type, row_id, data)
-
+                listener_refs = [row.id async for row in conn.execute(
+                    sa.select([
+                        listeners.c.id,
+                    ]).where(
+                        sa.and_(
+                            listeners.c.listen_on_type == resource_type,
+                            sa.or_(
+                                listeners.c.data.op('->')('notify_of_new') == 'true',
+                                sa.and_(
+                                    listeners.c.data.op('->')('notify_on_all') == 'true',
+                                    sa.or_(
+                                        listeners.c.data.op('->')('notify_of_new') == 'true',
```

Comment:
No. It is needed indeed, because listeners can have either
- notify_of_new = true
- notify_on_all = true ("notify_of_new" might not be set, therefore None)
- notify_on_all = true (but "notify_of_new" is set to false, thus no notification should be triggered)